src/containers/NotePop/NotePop.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+export class ErrorPop extends Component {
```

Comment:
@DevelopingDavid Please change the naming of the container to be the same as the class. Thanks!